validator/src/execution/py_executor.rs

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+use cpython;
+use cpython::ObjectProtocol;
+use cpython::PythonObject;
+use cpython::ToPyObject;
+
+use execution::execution_platform::ExecutionPlatform;
+
+use scheduler::Scheduler;
+use scheduler::py_scheduler::PyScheduler;
+
+pub struct PyExecutor {
+    pyobj: cpython::PyObject,
+}
+
+impl PyExecutor {
+    pub fn new(
+        context_manager: cpython::PyObject,
+        executor: cpython::PyObject,
+    ) -> Result<PyExecutor, cpython::PyErr> {
+        let py = unsafe { cpython::Python::assume_gil_acquired() };
+        Ok(PyExecutor {
+            pyobj: cpython::PyModule::import(py, "sawtooth_validator.execution.py_executor")
+                .unwrap()
```

Comment:
When unwrapping a PyResult like this, use `expect("helpful context")` to make debugging easier in the future. I think if you just unwrap, Rust just gives the line where unwrap is implemented in the std library instead of the line where the error occurred.

This comment also applies to other PyResult unwraps in this PR.

Reply:
My experience is that .unwrap() gives `ImportError: unable to import "sawtooth_validator.execution.py_executor"` which is at least as helpful as anything I would write. I think the `.expect("helpful content")` is helpful for reading through the code and understanding why unwrapping is ok. I will update to `expect` but because it allows for more understanding in the code. The debugging messages might be a wash.